### PR TITLE
Add gg138 v2

### PR DESCRIPTION
**Pull request by sjanssen2**
I have been wrong with loosing GG in #64 . However, I restructures the build.sh file such that the Silva references get imported into qza such that users can directly use those instead of the naked files. That will become handy for PR #63 and hopefully also solves the current build error.

### CODE REVIEW

File: ci/recipe/build.sh [hunk 1]
```diff
@@ -4,23 +4,34 @@ set -e
 set -x
 
 # extract Greengenes 13.8 reference files
+<<<<<<< HEAD
```
Comment (thermokarst) [inline ci/recipe/build.sh]: Looks like some diff cruft was left behind, can you please fix this up?

Comment (sjanssen2) [reply ci/recipe/build.sh]: @thermokarst should be solved by now

**Comment by sjanssen2** [comment]
@thermokarst and @ebolyen there is a new post in the forum https://forum.qiime2.org/t/problem-with-fragment-insertion-sepp/11396/6 showing the need to get this PR merged and delivered soon. Would be great if you can find some time to merge and see if it passes busywork.

**Comment by thermokarst** [comment]
Hi @sjanssen2! One of us will review soon! Just so you know though, the next release of QIIME 2 isn't until the end of October.

Busywork is still broken due to the packaging changes for this plugin, so that will need to be a higher priority. Thanks!

**Pull request merged**